testsuite/integration-tests/pom.xml

```diff
@@ -390,7 +394,13 @@
                         </execution>
                     </executions>
             </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-compiler-plugin</artifactId>
+                <configuration>
+                    <parameters>true</parameters>
```

Comment:
This line is not necessary, if we define "maven.compiler.parameters" property, right?

https://maven.apache.org/plugins/maven-compiler-plugin/compile-mojo.html

Reply:
I didn't know about this property. We can try it.

Reply:
@marekkopecky right, thanks, fixed